openreview/api/iThenticate_client.py

```diff
@@ -0,0 +1,233 @@
+import os
+import requests
+import json
+from datetime import datetime
+
+
+class iThenticateClient:
+    def __init__(self, api_key):
```

Comment:
@akshatchhabra I moved the client to this folder and added the api_key as a parameter. The venue that is calling to detect plagiarism should provide the api key.